frontend/src/libs/components/header/header.tsx

```diff
@@ -1,10 +1,16 @@
 import { AppRoute } from '~/libs/enums/enums.js';
 import { useCallback, useNavigate } from '~/libs/hooks/hooks.js';
+import { type BurgerMenuItem } from '~/libs/types/types.js';
 
-import { AppLogo, Button, Link } from '../components.js';
+import { AppLogo, BurgerMenu, Button, Link } from '../components.js';
 import styles from './styles.module.scss';
 
-const Header: React.FC = () => {
+type Properties = {
+  menuItems?: BurgerMenuItem[];
```

Comment:
```suggestion
  menuItems: BurgerMenuItem[];
```
can we make it required?